app/views/api_docs/vendor_api_docs/pages/release_notes.md

```diff
@@ -1,3 +1,9 @@
+## v1.5 — 16th December 2024
+
+Minor Version Upgrade:
+
+Release API version `v1.5` to production. Please refer to the [API Reference](/api-docs/v1.6/reference) for details.
+
 ## v1.5pre — 10th July 2024
```

Comment:
```suggestion
## v1.5 — 10th July 2024
```

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/10172#discussion_r1882866385

Reply:
There's a fix for this here btw: 

https://github.com/DFE-Digital/apply-for-teacher-training/pull/10184